docs/spec/is_bitwise_trivially_relocatable.md

```diff
@@ -0,0 +1,103 @@
+# Class template `is_bitwise_trivially_relocatable`
+
+> Header: `proxy.h`  
+> Module: `proxy`  
+> Namespace: `pro::inline v4`  
+> Since: 4.0.0
+
+```cpp
+template <class T>
+struct is_bitwise_trivially_relocatable;
+
+template <class T>
+constexpr bool is_bitwise_trivially_relocatable_v =
+    is_bitwise_trivially_relocatable<T>::value;
+```
+
+The class template `is_bitwise_trivially_relocatable<T>` is a type trait whose `value` is `true` when objects of (complete) type `T` can be *bitwise trivially relocated*: a new object of type `T` can be created at an arbitrary suitably aligned storage location by performing a raw byte-wise copy (as if by `std::memcpy`) of `sizeof(T)` bytes from the original object's storage, and the original object can then be considered destroyed (its lifetime ends) without invoking its destructor. Otherwise the `value` is `false`.
+
+Semantics follow the model described in [P3780R0](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3780r0.html), not the `std::trivially_relocatable` facility from [P2786R13](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p2786r13.html) (C++26). The library keeps this separate trait for portability and because its internal optimizations rely on the "memcpy relocation" guarantee.
+
+## Definition
+
+The primary template is defined as:
+
+```cpp
+template <class T>
+struct is_bitwise_trivially_relocatable
+    : std::bool_constant<std::is_trivially_move_constructible_v<T> &&
```

Comment:
Does this also consider https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p2786r13.html#do-security-technologies-like-pointer-authentication-mean-that-polymorphic-types-cant-be-trivially-relocatable ?